assembl/scripts/ini_files.py

```diff
@@ -223,12 +223,10 @@ def populate_random(random_file, random_template=None, saml_info=None):
     from os import urandom
     from assembl.auth.make_saml import make_saml_key, make_saml_cert
     base = Parser()
-    if random_template is None:
-        random_template = os.path.basename(random_file) + ".tmpl"
```

Comment:
Why not keep these 2 lines? except we change random_template to random_templates, like so:
```
if random_templates is None:
    random_templates = os.path.basename(random_file) + ".tmpl"
```